_posts/2022-11-07-what-makes-a-python-package-healthy.md

```diff
@@ -0,0 +1,559 @@
+
+### Snyk and security (which aren't discussed in this post)
+One topic that I am not delving into in this post is security issues. Snyk 
+is definitely a leader in this space and was mentioned at least once in 
+the conversation. 
+
+### Some existing metric examples 
+
+Below are some of the metrics that you can easily access via Snyk's website. 
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">This might be helpful. This website collects various metrics. And here is the example for numpy. <a href="https://t.co/YNsRoMgks4">https://t.co/YNsRoMgks4</a></p>&mdash; Kevin Wang (@KevinWangStats) <a href="https://twitter.com/KevinWangStats/status/1577999823911161856?ref_src=twsrc%5Etfw">October 6, 2022</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script> 
+
+
+
+#### [Pandera python package metrics on Snyk](https://snyk.io/advisor/python/pandera)
+
+<figure>
+    <a href="/images/pyopensci-packages/snyk-pandera-python-package-metrics.png"><img src="/images/pyopensci-packages/snyk-pandera-python-package-metrics.png" alt="Image showing the metrics from the Snyk website for the `pandera` package in our ecosystem."></a>
+    <figcaption>Here you can see what a Snyk report looks like for pandera, a package accepted into our ecosystem a few years ago. Pandera gets a very healthy report because it's heavily used among other metrics.
+    </figcaption>
+</figure>
+
+#### Now let's look at [pyGMT package statistics on Snyk](https://snyk.io/advisor/python/pygmt)
+
+{% include figure image_path="/images/pyopensci-packages/snyk-pygmt-python-package-metrics.png" alt="Image showing the metrics from the snyk website for the pyGMT package in our ecosystem" caption="In comparison to pandera, pyGMT gets a lower health score. I suspect this is due to lower community adoption and use. pyGMT is a much newer package. We'd argue however that pyGMT has a very healthy level of maintenance and even healthier package structure." %}
+
+<figure>
+    <a href="/images/pyopensci-packages/snyk-pygmt-python-package-metrics.png"><img src="/images/pyopensci-packages/snyk-pygmt-python-package-metrics.png" alt="Image showing the metrics from the snyk website for the pyGMT package in our ecosystem."></a>
+    <figcaption>In comparison to pandera, pyGMT gets a lower, but still good, health score. I suspect this is due to lower community adoption and use. pyGMT is a much newer package. We'd argue however that pyGMT has a very healthy level of maintenance and even healthier package structure.
+    </figcaption>
+</figure>
+
+And of course the [scientific Python project has also been tracking the larger packages](https://devstats.scientific-python.org/):
```

Comment:
did you mean for there to be an image here? Since the sentence ends with a colon

Might be nice too include an image from the link (if you weren't already planning on it)
e.g., the scipy one that lives at https://devstats.scientific-python.org/_images/scipy.png

Reply:
no this is just a link to see the live stats.